app/src/main/java/com/cmput301f20t21/bookfriends/ui/borrow/accepted/MeetLocationDialog.java

```diff
@@ -0,0 +1,78 @@
+package com.cmput301f20t21.bookfriends.ui.borrow.accepted;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+
+import com.cmput301f20t21.bookfriends.R;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapView;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.firebase.firestore.GeoPoint;
+
+public class MeetLocationDialog  extends DialogFragment implements OnMapReadyCallback {
+    private GoogleMap map;
+    private MapView mapView;
+    private GeoPoint geoPoint;
+
+    private Button cancelButton;
+
+    public MeetLocationDialog(GeoPoint point) {
+        this.geoPoint = point;
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
+        Dialog dialog = super.onCreateDialog(savedInstanceState);
+        dialog.setTitle("View Meet Location");
```

Comment:
one more here, and it looks like the dialog don't have a title? I'm not sure about this